src/Jackett.Common/Definitions/hdfans.yml

```diff
@@ -0,0 +1,179 @@
+---
+id: hdfans
+name: HDFans
+description: "HDFans is a CHINESE Private Torrent Tracker for HD MOVIES / TV"
+language: zh-CN
+type: private
+encoding: UTF-8
+links:
+  - https://hdfans.org/
+
+caps:
+  categorymappings:
+    - {id: 401, cat: Movies, desc: "Movies/电影"}
+    - {id: 404, cat: TV/Documentary, desc: "Documentaries/纪录片"}
+    - {id: 405, cat: TV/Anime, desc: "Animations/动漫"}
+    - {id: 402, cat: TV, desc: "TV Series/剧集"}
+    - {id: 403, cat: TV, desc: "TV Shows/综艺"}
+    - {id: 404, cat: Other, desc: "Education/教育"}
+    - {id: 406, cat: Audio/Video, desc: "MusicVideo/音乐MV"}
+    - {id: 407, cat: TV/Sport, desc: "Sports/运体"}
+    - {id: 409, cat: Other, desc: "Misc/其他"}
+    - {id: 408, cat: Audio, desc: "HQ Audio/无损音乐"}
+    - {id: 419, cat: Other, desc: "Software/软件"}
+    - {id: 421, cat: Other, desc: "Games/游戏"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep, imdbid, doubanid]
+    movie-search: [q, imdbid, doubanid]
```

Comment:
Is doubanid searching supported? If so, you'd need to implement it in the search block, e.g. - https://github.com/Jackett/Jackett/blob/master/src/Jackett.Common/Definitions/52pt.yml

Reply:
it isn't support douban id .it only support imdbid to search

Reply:
Ok, then you can just remove doubanid searching.